package.json

```diff
@@ -64,6 +67,8 @@
     "react-dnd-touch-backend": "lock this to 0.2.7 to support dnd on mobile - see https://github.com/yahoo/react-dnd-touch-backend/issues/34",
     "react-dnd-touch-backend": "0.2.7",
     "react-highcharts": "~11.5.0",
+    "react-router": "~2.8.0",
```

Comment:
Yeah there shouldn't be too much fallout in 3 and it should catch all of the proptype warnings. v4 on the other hand is a big jump and refactor